client/src/components/Sharing/Embeds/WorkflowEmbed.vue

```diff
@@ -184,8 +184,8 @@ const clipboardTitle = computed(() => (copied.value ? "Copied!" : "Copy URL"));
     .settings {
         flex: 1;
         display: flex;
-        align-items: start;
-        justify-content: start;
+        align-items: flex-start;
+        justify-content: flex-start;
```

Comment:
Thank you for the performance improvements. I'm a bit confused by these changes though. Are they related to the PR?

Reply:
These were addressing warnings from postcss that were emitted during the build process and became much more obvious/annoying with the caching in place.

Realistically we likely already don't support browsers that would lack universal `end` support and this would be a problem in, but for instances like this, in an intentional flex layout, I figured it was worth it to just be explicit.

Reply:
Thank you for explaining!